### ISSUE TITLE
Add type predicates for series type

### ISSUE THREAD

**Issue by SuperPenguin** [tradingview/lightweight-charts#670]
**Is your feature request related to a problem? Please describe.**

I need to narrow down my series type from collection of `ISeriesApi<SeriesType>` in typescript

**Describe the solution you'd like**

`isLineSeries(series)`, etc.

**Comment by SuperPenguin** [comment]
I think this could be added to `series-api.ts`

```ts
export class SeriesApi<TSeriesType extends SeriesType> implements ISeriesApi<TSeriesType> {
    public seriesType(): SeriesType {
        return this._series.seriesType();
    }
}

export function isLineSeries(series: ISeriesApi<SeriesType>): series is ISeriesApi<'Line'> {
    return series.seriesType() === 'Line';
}

// Usage
if (isLineSeries(series)) {
    // is line series
}
```

or 

```ts
export class SeriesApi<TSeriesType extends SeriesType> implements ISeriesApi<TSeriesType> {
    public isLineSeries(): this is ISeriesApi<'Line'> {
        return this._series.seriesType() === 'Line';
    }
}

// Usage
if (series.isLineSeries()) {
    // is line series
}
```

Honestly, I prefer the first one

**Comment by timocov** [comment]
> Honestly, I prefer the first one

Agree.

`public seriesType(): SeriesType` I like this one, but I'm not sure that `export function isLineSeries` should be in the library, it might be easily added to the customer's code if needed, but I don't think that we need to add this function to the bundle. Perhaps we need to add some "utils" additional bundle with such helpers functions though but we don't have that one.

So I think we can start from adding a method to `ISeriesApi` with returning current series' type (just because you can't get it now in any way) and then see whether additional type guards is required to be in the library.

**Comment by timocov** [comment]
The fix is just published in [3.3.0 version](https://github.com/tradingview/lightweight-charts/releases/tag/3.3.0).
